base.js

```diff
@@ -78,6 +78,7 @@ module.exports = {
         '@typescript-eslint/no-loop-func': 'error',
         'no-redeclare': 'off',
         '@typescript-eslint/no-redeclare': 'error',
+        '@typescript-eslint/no-redundant-type-constituents': 'off',
```

Comment:
- Please move the rule higher and sort alphabetically. This section is specific to overrides of the base eslint rules.
- Please explain why it's disabled in a comment above the rule.

Reply:
Hmm, sort seems good to me `redeclare < redundant < unused`.
Ok I'll explain in comments (It's in message commit, but yes it's not the best place)

Reply:
Sort is ok within this section, but this is the wrong section.

Reply:
Ok,
I added comment section with `//region` and `//endregion` syntax